service-python/assessclaimdc6602v2/build.gradle

```diff
@@ -0,0 +1,23 @@
+plugins {
+    id 'starter.java.container-conventions'
+    id 'starter.java.build-utils-conventions'
+}
+
+docker { // expects src/docker/Dockerfile to exist
+    // Copy files into build/docker/ folder, where the container image will be built
+    copySpec.from("src").into("."){ 
+        exclude('docker')
+    }
+    copySpec.into(".").from("../"){
+        include("*.py")
+    }
+    // Use container image cache
+    noCache false
+}
+
+dockerRun {
+    image "${dockerRegistry}/${rootProject.name}-service-assessclaimdc6602"
+    env 'RABBITMQ_PLACEHOLDERS_USERNAME': getEnvOrDefault('DB_SERVICE_PASSWORD', 'also-not-the-service-password'),
```

Comment:
I do not think DB_SERVICE_PASSWORD is the right env variable. We should match whatever docker-compose is using including the defaults.

Reply:
This file has been changed